projects/boringssl/fuzz_certs.cc

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
assuming these fuzz targets are testing the same APIs as the existing plan data fuzz targets, it might be useful to have a comment here saying "This fuzz targets is similar to <link_to_the_existing_fuzz_target_in_boringssl_repo>, but it employs libprotobuf-mutator for structure-aware fuzzing", something along these lines

Reply:
same comment for the other targets.